src/Plivo/HttpClients/PlivoGuzzleHttpClient.php

```diff
@@ -101,7 +101,10 @@ public function send_request($url, $method, $body, $headers, $timeOut, $request)
         $headers["Authorization"] = "Basic " . base64_encode("$this->authId:$this->authToken");
         $request->setHeaders($headers);
         $options =[];
-        $requestBody = json_encode($request->getParams(), JSON_FORCE_OBJECT);
+        $requestBody = json_encode($request->getParams());
+        if(empty($request->getParams())){
```

Comment:
if  $request->getParams() is empty ? what is the point of doing json_encode($request->getParams(), JSON_FORCE_OBJECT); ?

Reply:
@narayana-plivo , JSON_FORCE_OBJECT converts an empty PHP Array to empty json -> {}. Normally, for non empty PHP Arrays, json_encode converts it to a proper json and sends it to the backend whereas in cases of empty arrays it goes to the backend as [] instead of {} and hence the change

Reply:
incase it didnt match the if condition, it wont break in line number 110? Undefined variable

Reply:
no, it wouldn't break bcoz if it doesn't match the if condition, the code on line 104 takes care of `$requestBody` variable and hence won't be undefined